main/ffnvcodec-headers/template.py

```diff
@@ -0,0 +1,14 @@
+pkgname = "ffnvcodec-headers"
```

Comment:
what's up with this scuffed name

why not just nv-codec-headers

Reply:
idk it's been a while, i don't remember why i chose this name

Reply:
probably from copying another distro and cause i called it that

Reply:
it's all YOUR fault!!11!!!1

Reply:
😔